packages/theming/demo/utilities.stories.mdx

```diff
@@ -18,6 +19,21 @@ import README from '../README.md';
 
 # Demo
 
+## PALETTE
+
+<Canvas>
+  <Story
+    name="PALETTE"
+    args={{ palette: PALETTE }}
```

Comment:
Should we let the user interact with the arguments for this story? If so, the current implementation will only track changes to the outer color keys. That's because `Hue` gets the colors from the [theme.palette](https://github.com/zendeskgarden/react-components/pull/1795/files#diff-fe006ad1caedbe3ffabf042e74a6e9a5b8f3d63ad21cd74525f7521381442877R74) and not the palette object injected as an argument. 

![Screen Shot 2024-04-22 at 12 58 56 PM](https://github.com/zendeskgarden/react-components/assets/6879688/62d36514-41ac-41a4-9251-f58857f5b2a5)

Reply:
I'm not super concerned about whether or not the story is interactive – more interested in demonstrating where the values are coming from. However, there is some copy/paste jank in here from the `website` implementation. Now cleaned up. 🧹